clients/js/src/constants.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  API: {
+    INFO: '/info',
+    SERVICES: '/services',
+    TRAIN: '/train',
```

Comment:
If we don't have support for train within the client at the moment, let's not put it in here.